teknologiradar.json

```
@@ -46,10 +46,16 @@
     {
       "key": "Google BigQuery",
       "title": "Google BigQuery",
-      "ring": "vurder",
+      "ring": "bruk",
       "quadrant": "infrastruktur",
       "description": "Populær og superskalerbar databaseteknologi for lagring av data. Kan også brukes til klargjøring og utforsking av data. Egner seg først og fremst for OLAP (Online Analytical processing) i motseting til OLTP (Online transaction processing). For sistnevnte bør man vurdere andre teknologier. BQ støtter SQL og kan kombineres med flere tjenester fra Google, noe som kan være en fordel i enkelte situasjoner. Mer info her: https://cloud.google.com/bigquery",
       "timeline": [
+        {
+          "moved": 0,
```

```suggestion
          "moved": 1,
```
Siden det ble flyttet 1 innover 🚀